src/loaders/zarrLoader.js

```diff
@@ -89,9 +124,12 @@ export default class ZarrLoader {
     return [data];
   }
 
-  async getRaster({ z, level }) {
+  async getRaster({ z }) {
     const source = this.isPyramid ? this._data[z] : this._data;
-    const { data } = await source.getRaw([level, null, null]);
+    const selection = [...this.chunkIndex];
```

Comment:
Any reason to not mirror `chunkKey` name from above?

Reply:
Technically the raster is an array selection rather than a single chunk. Chunk keys specify the key for a particular zarr chunk, whereas an array selection might include several chunks.